bindings/java/mongocrypt/build.gradle.kts

```diff
@@ -42,7 +42,7 @@ repositories {
 }
 
 group = "org.mongodb"
-version = "1.0.0-beta2"
+version = "1.0.0-SNAPSHOT"
```

Comment:
I'm not sure what the convention here is. If this is meant to track the libmongocrypt version, than perhaps this should remain beta2 until we tag libmongocrypt with the beta3 tag?

Reply:
I just wanted a SNAPSHOT to be published on the snapshots repo until the Java driver work is done. Once thats done a beta3 sounds good.